documentation/src/main/asciidoc/topics/config_examples/jgroups_asym_encrypt.xml

```diff
@@ -0,0 +1,22 @@
+<infinispan>
+    <jgroups>
+         <stack name="encrypt-tcp" extends="tcp"> <1>
+           <SSL_KEY_EXCHANGE keystore_name="mykeystore.jks" <2>
+                             keystore_password="changeit" <3>
+                             stack.combine="INSERT_AFTER"
+                             stack.position="VERIFY_SUSPECT"/> <4>
+           <ASYM_ENCRYPT asym_keylength="512" <5>
+                    asym_algorithm="RSA" <6>
+                    change_key_on_coord_leave = "false" <7>
+                    change_key_on_leave = "false" <8>
+                    use_external_key_exchange = "true" <9>
+                    stack.combine="INSERT_BEFORE"
```

Comment:
@tristantarrant I'd like to backport this PR to 11 but INSERT_BEFORE isn't available. I'm not sure how to use stack.combine to get SSL_KEY_EXCHANGE and ASYM_ENCRYPT in the stack before pbcast.NAKACK2. Any recommendations?

Reply:
Well, since our stacks all have `VERIFY_SUSPECT` above `pbcast.NAKACK2` just use `INSERT_AFTER` with that. Not ideal, but saves us from backporting

Reply:
Right but the problem I've found is that I can't use `INSERT_AFTER` with both `SSL_KEY_EXCHANGE` and `ASYM_ENCRYPT`.

Reply:
Actually it looks like it's OK if I do this (on 11):

```
<SSL_KEY_EXCHANGE ... stack.combine="INSERT_AFTER" stack.position="VERIFY_SUSPECT"/>
<ASYM_ENCRYPT ... stack.combine="INSERT_AFTER" stack.position="SSL_KEY_EXCHANGE"/>
```

That seems to work fine. I'll update with that config when I create the backport for this content to 11 (unless you see anything wrong with it). Thanks.